ui/src/qd.ts

```diff
@@ -807,9 +809,25 @@ export const qd: Qd = {
       }
     return { get, put, set, del, drop, sync }
   },
-  sync: () => {
+  sync: async () => {
```

Comment:
2 reasons:
* When should be the files submitted? It has to be submitted before sync AFAIU so that args can contain references to uploaded files.
* How would I sync busy overlay with both files upload and long-running sync?

Reply:
Can't you perform the upload in the file-upload component, wait for the upload to complete, and then call `sync()` instead of implementing the file-upload logic inside `sync()`? 

To show/hide the busy overlay, you can call `qd.busyB(true)`/`qd.busyB(false)` from anywhere.